acceptance/tests/external_facts/user_uses_default_external_fact_dir.rb

```diff
@@ -0,0 +1,54 @@
+test_name "Non-root user default external facts directory (facts.d) is searched for facts"
+
+require 'facter/acceptance/user_fact_utils'
+extend Facter::Acceptance::UserFactUtils
+
+#
+# This test is intended to ensure that executable external facts placed into the
+# default facts.d directory are properly found and resolved by Facter when run
+# as a user other than root. In such a case, facts.d is found in the user's homedir.
+#
+
+user_name = 'testuser'
+unix_content = <<EOM
+#!/bin/sh
+echo "external_fact=testvalue"
+EOM
+
+win_content = <<EOM
+echo "external_fact=testvalue"
+EOM
+
+agents.each do |agent|
+  if agent['platform'] =~ /windows/
+    content = win_content
+  else
+    content = unix_content
+  end
+
+  step "Agent #{agent}: create new user"
+  on(agent, "useradd #{user_name} --home /home/#{user_name}")
+  on(agent, "mkdir -p /home/#{user_name}")
+  on(agent, "chown #{user_name}:#{user_name} /home/#{user_name}")
```

Comment:
@joshcooper hm, but running Facter acceptance standalone means Puppet isn't present. I tried the Puppet route earlier and ran into that